pkgs/servers/gotify/default.nix

```diff
@@ -35,11 +36,14 @@ buildGoModule rec {
   ui = callPackage ./ui.nix { };
 
   preBuild = ''
-    cp -r ${ui}/libexec/gotify-ui/deps/gotify-ui/build ui/build && go run hack/packr/packr.go
+    if [ -n "$ui" ]; then
```

Comment:
Why did you add this conditional?

Reply:
I added a comment:

> to make the preBuild a no-op inside the goModules fixed-output derivation, where it would fail

Reply:
> Why did you add this conditional?

I would have never have imagined. Thanks for that comment!